bullet/src/SDFFeatures.cc

```diff
@@ -42,7 +42,7 @@ Identity SDFFeatures::ConstructSdfModel(
   // Build links
   for (std::size_t i = 0; i < _sdfModel.LinkCount(); ++i)
   {
-    this->ConstructSdfLink(modelIdentity, *_sdfModel.LinkByIndex(i));
+    this->FindOrConstructSdfLink(modelIdentity, *_sdfModel.LinkByIndex(i));
```

Comment:
I have confirmed this loop is never entered, as models get constructed without links. We should delete it to save misunderstandings

Reply:
I tested removing it and `shapes.sdf` still works. So I removed it with b8758a8. 

This looks odd to me, at least tthrough the whole `dartsim` implementation they seemed to create the links there.